doc/subsystems/networking/overview.rst

```diff
@@ -47,6 +47,10 @@ can be disabled if not needed.
   and client roles can be used the the application. The amount of TCP sockets
   that are available to applications can be configured at build time.
 
+* **BSD Sockets API** There is an experimental support for a BSD Sockets
```

Comment:
Experimental support for a subset of a compatible BSD Sockets API is implemented.